### DISCUSSION TITLE
Request for Options for Price Checking Items

### DISCUSSION THREAD

**Discussion by DraconisMaximus** [SnosMe/awakened-poe-trade#766] [Ideas]
Hi, I'm not sure the process for this or if this is the correct place to ask but I'd like to request the ability to set the price check to search only for chaos, divine or even whatever you want to set it to if that's not too much of a problem.  This is to avoid all of the weird exalt, awakened sextant, etc prices that serve only to confuse and take advantage of people.  I would also like the option added to search only uniques by exact roll.  Thanks for your time, I love the app.  It improves my experience with the game greatly.

**Comment by SnosMe** [comment]
> ability to set the price check to search only for chaos, divine

Wait for new release
https://github.com/SnosMe/awakened-poe-trade/issues/753#issuecomment-1272308099

> search ONLY uniques by exact roll

Why?

**Comment by DraconisMaximus** [comment]
Unique modifiers make much bigger of a difference with exalt/divine change.  Thanks for reply
